jobs/Scripts/simpleRender.py

```diff
@@ -1,67 +1,145 @@
+		exit(-1)
 
-	work_dir = args.output
 
-	with open(os.path.join(os.path.dirname(sys.argv[0]), "Templates", "base_function.py")) as f:
-		base = f.read()
+	core_config.main_logger.info('Make script')
 
-	with open(os.path.join(os.path.dirname(sys.argv[0]), args.template)) as f:
-		blender_script_template = f.read()
+	try:
+		with open(os.path.join(os.path.dirname(__file__), "base_functions.py")) as f:
+			script = f.read()
+		with open(os.path.realpath(os.path.join(os.path.dirname(__file__),  '..', 'Tests', args.testType, 'test_cases.json'))) as f:
+			test_cases = f.read()
+	except OSError as e:
+		core_config.main_logger.error(str(e))
+		return 1
+
+	try:		
+		with open(os.path.join(os.path.dirname(__file__), 'extensions', args.testType + '.py')) as f:
+			extension_script = f.read()
+		script = script.split('# place for extension functions')
+		script = script[0] + extension_script + script[1]
+	except:
+		pass
 
-	blender_script_template = base + blender_script_template
+	res_path = args.res_path
+	work_dir = os.path.abspath(args.output)
+	script = script.format(work_dir=work_dir, testType=args.testType, render_device=args.render_device, res_path=res_path,
+							  resolution_x=args.resolution_x, resolution_y=args.resolution_y, SPU=args.SPU)
 
-	blenderScript = blender_script_template.format(work_dir=work_dir, render_mode=args.render_mode, resource_path=args.resource_path, testCases=args.testCases,\
-													resolution_x=args.resolution_x, resolution_y=args.resolution_y, max_samples=args.max_samples, \
-													SPU=args.SPU, package_name=args.package_name)
+	with open(os.path.join(args.output, 'base_functions.py'), 'w') as file:
+		file.write(script)
 
-	blenderScriptPath = os.path.join(work_dir, 'script.py')
-	with open(blenderScriptPath, 'w') as f:
-		f.write(blenderScript)
+	try:
+		cases = json.load(open(os.path.realpath(
+			os.path.join(work_dir, 'test_cases.json'))))
+	except:
+		try:
+			cases = json.load(open(os.path.realpath(os.path.join(os.path.dirname(__file__),  '..', 'Tests', args.testType, 'test_cases.json'))))
+		except:
+			core_config.logging.error('Problem with test_cases.json')
 
-	cmdRun = '"{tool}" -b -P "{template}"\n'.format(tool=args.tool, template=blenderScriptPath)
+	try:
+		with open(os.path.join(os.path.dirname(__file__), args.testCases)) as f:
+			tc = f.read()
+			testCases = json.loads(tc)[args.testType]
+		temp = []
+		for case in cases:
+			if case['case'] in testCases:
+				temp.append(case)
+		cases = temp
+	except:
+		pass
+
+	core_config.main_logger.info('Create empty report files')	
+
+	if not os.path.exists(os.path.join(work_dir, 'Color')):
+		os.makedirs(os.path.join(work_dir, 'Color'))
+	copyfile(os.path.abspath(os.path.join(work_dir, '..', '..', '..', '..', 'jobs_launcher',
+                                       'common', 'img', 'error.jpg')), os.path.join(work_dir, 'Color', 'failed.jpg'))
+
+	temp = [platform.system()]
+	temp.append(get_gpu())
+	temp = set(temp)
```

Comment:
```suggestion
    render_device = get_gpu()
    current_conf = set(platform.system()) if not render_device else {platform.system(), render_device}
```

FYI: get_gpu() возвращает false, если не смог определить карту